code/modules/mob/living/carbon/human/human_damage.dm

```diff
@@ -107,6 +107,12 @@
 		wearing_rig.notify_ai("<span class='danger'>Warning: user consciousness failure. Mobility control passed to integrated intelligence system.</span>")
 	..()
 
+/mob/living/carbon/human/update_canmove()
+	var/old_lying = lying
+	. = ..()
+	if(lying && !old_lying && !resting && !buckled) // fell down
+		playsound(loc, species.bodyfall_sound, 50, 1, -1)
```

Comment:
How does this even work, bodyfall_sound is a string

Reply:
I tested it and it works

I guess it's a little confusing since bodyfall_sound is used twice